lib.rs

```diff
@@ -315,6 +315,29 @@ impl<A: Array> SmallVec<A> {
     }
 }
 
+impl<A: Array> SmallVec<A> where A::Item: Copy {
+    pub fn insert_slice(&mut self, index: usize, slice: &[A::Item]) {
+        self.reserve(slice.len());
```

Comment:
I believe reserve takes in an "additional" amount required and ensures the capacity is large enough, so I think this is ok.